app/controllers/course/assessment/question/text_responses_controller.rb

```diff
@@ -6,12 +6,16 @@ class Course::Assessment::Question::TextResponsesController < \
                               through: :assessment, parent: false
 
   def new
+    if params[:file_upload] == 'true'
+      @text_response_question.hide_text = true
```

Comment:
Should we use a positive boolean instead? `show_text` instead of `hide_text`. Seems easier to reason about. Like how ben wants us to change draft? to published?